src/components/OrderHistoryDetail/OrderHistoryDetailItem.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components';
+import { Common } from '@styles/globalStyle';
+
+import Button from '@components/common/Button';
+
+interface Props {
+  deliveryName: string;
+  price: number;
+  isPayed: boolean;
+}
+
+const OrderHistoryDetailItem = ({ deliveryName, price, isPayed }: Props) => {
+  const stringPrice = price.toLocaleString('ko-KR');
+  return (
+    <ListItem>
+      <InfoWrapper>
+        <CheckBox type="checkbox"></CheckBox>
+        {isPayed ? (
+          <Text>
+            '{deliveryName}'님이 {stringPrice}P 결제 완료했습니다.'
+          </Text>
```

Comment:
아래처럼 작성하지 않아도 잘 표시되나요..?

```suggestion
          <Text>
            {`${deliveryName}'님이 ${stringPrice}P 결제 완료했습니다.`}
          </Text>
```

Reply:
문제없이 잘 표시 됩니다!